source/modules/collection-api/event/overview.md.erb

```diff
@@ -1,21 +1,21 @@
-
-- <a href="#event-creation">trackEventCreation</a>
-- <a href="#event-updates">trackEventUpdate</a>
-- <a href="#event-deletion">trackEventDeletion</a>
-
-All functions for tracking event activity require an [InspetorEvent](#inspetorevent) object as argument and return `true` upon successful execution. Otherwise, one of the following exceptions will be thrown:
-
-Exception | Description
---------- | -----------
-**[EventException](#inspetoreventexception)**       | The Event object provided is invalid
-**[AddressException](#inspetoraddressexception)**   | The Address object provided as a property of the Event object is invalid
-**[generalException](#inspetorgeneralexception)** | The timestamps passed as Event object properties are invalid
-**[TrackerException](#trackerexception)**   | An internal error occurred (*This should never happen*)
+You can send event-related information to Inspetor via the REST API. The `/events` endpoint supports `POST` (create), `PUT` (update), and `DELETE` (delete) operations with the following arguments:
 
+Property        | When is it required?   | Type    | Description
+--------        | ---------------------  | ----    | -----------
+**id**          | `POST`, `PUT`, `DELETE`| String  | The unique identifier of this event listing in your platform
+**name**        | `POST`                 | String  | The name of the event
+**is_physical** | `POST`                 | Bool    | It determines if it is a physical or digital event. *The default value is `true`.*
+**sessions**    | `POST`                 | Array of Inspetor [event datetimes](#event-datetimes) | All available dates and times that the event occurs (if it has more than one) and a unique id for each date/time.
+**creator_id**  | `POST`                 | String  | The id of the user who created the event
+**admin_ids**   | `POST`                 | Array   | An array containing the ids of all users who can make changes to the event
+address         |                        | [Address](#addresses) | The address where the event is happening. *This field is only required if the type of the event is set to physical.*
+status          |                        | String  | The status of the event. *Inspetor provides some pre-built statuses (`draft`, `private` and `published`) but you can provide a custom status if it does not fit with our status model.* 
+description     |                        | String  | A description of the event
+slug            |                        | String  | The slug of the event
 
 <p><%= partial "modules/collection-api/event/actions/create.md.erb" %></p>
 <p><%= partial "modules/collection-api/event/actions/update.md.erb" %></p>
-<p><%= partial "modules/collection-api/event/actions/delete.md.erb" %></p>
\ No newline at end of file
+<p><%= partial "modules/collection-api/event/actions/delete.md.erb" %></p>
+<p><%= partial "modules/collection-api/event/sessions.md.erb" %></p>
```

Comment:
I think you have to update
<p><%= partial "modules/collection-api/event/sessions.md.erb" %></p>
to
<p><%= partial "modules/collection-api/event/event_datetimes.md.erb" %></p>

nit: EOF